knees4bees/overlook
# Featuer/login

## What does this PR do? ##
- Adds login functionality with validation.
- Tweaks styling.
- Cleans up code.

## How should it be tested? ##
- Confirm that app meets project spec requirements.
- Confirm that all tests are still passing.
